resource-managers/kubernetes/integration-tests/README.md

```diff
@@ -199,9 +199,9 @@ to the wrapper scripts and using the wrapper scripts will simply set these appro
   <tr>
     <td><code>spark.kubernetes.test.javaImageTag</code></td>
     <td>
-      A specific OpenJDK base image tag to use, when set uses it instead of azul/zulu-openjdk.
+      A specific Azul Zulu OpenJDK base image tag to use, when set uses it instead of 21.
```

Comment:
Thank you for fixing this.

Reply:
I think the comment is right. I guess the error will not occur after this PR.
```
$ build/sbt -Pkubernetes -Pkubernetes-integration-tests \
  -Dspark.kubernetes.test.namespace=default \
  -Dtest.exclude.tags=local,r,minikube \
  "kubernetes-integration-tests/test"
```
This should works good.